server/src/server.js

```diff
@@ -0,0 +1,390 @@
+import express from "express";
+import mongoose from "mongoose";
+import bodyParser from "body-parser";
+import cors from "cors";
+
+import User from "./models/user.models.js";
+import Insurance from "./models/insurances.model.js";
+
+const connectionString = "mongodb://localhost:27017/fin2u";
```

Comment:
Please refactor the server.js: Create routes and controllers and then I'll take a look again ;)